src/es256.c

```diff
@@ -275,6 +302,28 @@ es256_pk_to_EVP_PKEY(const es256_pk_t *k)
 		goto fail;
 	}
 
+#if OPENSSL_VERSION_NUMBER >= 0x30000000
+	if ((g = EC_GROUP_new_by_curve_name(nid)) == NULL ||
+	    (q = EC_POINT_new(g)) == NULL ||
+	    EC_POINT_set_affine_coordinates(g, q, x, y, bnctx) == 0 ||
+	    (point_len = EC_POINT_point2buf(g, q, POINT_CONVERSION_UNCOMPRESSED, &pbuf, bnctx)) != 0)
```

Comment:
this should probably be point_len == 0. don't worry about fixing/pushing; i'll do the change when i pick up the chunk. just sharing in case i'm mistaken.

Reply:
I don't remember, maybe even <=0. Sorry, my fault.